packages/framework/src/Command/CronCommand.php

```diff
@@ -162,7 +189,10 @@ private function runCron(InputInterface $input, CronFacade $cronFacade, MutexFac
      */
     private function getCurrentRoundedTime()
     {
-        $time = new DateTime(null);
+        $cronTimezone = $this->parameterBag->get('shopsys.cron_timezone');
+        $cronTimezone = $cronTimezone ?? date_default_timezone_get();
+
+        $time = new DateTime(null, new DateTimeZone($cronTimezone));
```

Comment:
```suggestion
        $time = new DateTime('now', new DateTimeZone($cronTimezone));
```

https://www.php.net/manual/en/datetime.construct.php